api/stream.go

```diff
@@ -593,12 +582,15 @@ func (r streamRoutes) deleteVideoSection(c *gin.Context) {
 		return
 	}
 
-	go func() {
-		err := DeleteVideoSectionImage(r.DaoWrapper.WorkerDao, file.Path)
-		if err != nil {
-			log.WithError(err).Error("failed to generate video section images")
-		}
-	}()
+	file, err := r.FileDao.GetFileById(fmt.Sprintf("%d", old.FileID))
```

Comment:
IMO first, the error returned by `GetFileById` should be checked (and handled, e.g., with `log.WithError`). Then, `DeleteVideoSectionImage` should be executed when there is `file != nil`.
Maybe we could also use a boolean, and then have an `OK`. I could see this being a bit error prone and possibly ignoring other errors that `GetFileById` could return.

Reply:
Great input. Added a `!gorm.ErrNotFound` check.

Reply:
Ah, yes, that's the perfect fit here.